configuration/concepts/reviewofsystems.csv

```diff
@@ -0,0 +1,75 @@
+fd55467a-6453-4fc1-a9a7-0ff78e9baa4e,,,Tongue,Langue,Tongue,Langue,,,Misc,N/A,,,,,,,,,,,
+08887d3e-78fe-4dff-84e4-3eea4287df02,,,Pallet,Palais,Pallet,Palais,,,Misc,N/A,,,,,,,,,,,
+bcda6677-244d-466c-9c33-98c02855b990,,,Mouth and Throat Deformation,Déformation de la bouche et de la gorge,Mouth and Throat Deformation,Déformation de la bouche et de la gorge,,,Misc,Coded,,Langue;Palais,,,,,,,,,
+55887656-fb44-4fcc-a9bc-4d73101ed31c,,,Mouth and Throat,Bouche et de la gorge,Mouth and Throat,Bouche et de la gorge,,,Misc,Coded,,Douleur;Dysphagie (R13);Autre,,,,,,,,,
+1a705716-d86c-427f-9bb4-6487fe748988,,,Mouth and Throat Set,Bouche et de la gorge set,Mouth and Throat Set,Bouche et de la gorge set,,,ConvSet,N/A,,,Dents;Déformation de la bouche et de la gorge;Bouche et de la gorge;Autres observations,,,,,,,,
+c1eef44a-42ae-41a1-a78f-26a354c59334,,,Eyes,Yeux,Eyes,Yeux,,,Misc,Coded,,Trouble visuel;Douleur;Conjonctivite;Prurit yeux;Écoulement des yeux;Larmoiement;Pâleur;Autre,,,,,,,,,
+bb9bf90d-1fad-4240-a79d-dbd6fcc1e038,,,Eyes Set,Ensemble d'yeux,Eyes Set,Ensemble d'yeux,,,ConvSet,N/A,,,Yeux;Autres observations,,,,,,,,
+4463d07e-fa84-4f31-8bb1-afc1d04a4501,,,Ears,Oreilles,Ears,Oreilles,,,Misc,Coded,,Acouphènes;Vertiges;Surdité;Écoulement;Douleur;Autre,,,,,,,,,
+85bd41e4-5982-4ffe-b0c3-92fcc6c7476b,,,Ears Set,Ensemble d'oreilles,Ears Set,Ensemble d'oreilles,,,ConvSet,N/A,,,Oreilles;Autres observations,,,,,,,,
+669eb2ed-6fe3-47d0-b9ed-17e3b8af1e85,,,Nose,Nez,Nose,Nez,,,Misc,Coded,,Saignements;Nez Sécrétions;Congestions des sinus;Autre,,,,,,,,,
+b970d95f-ebc7-4fcb-b163-cd8be32328e3,,,Nose Set,Ensemble de nez,Nose Set,Ensemble de nez,,,ConvSet,N/A,,,Nez;Autres observations,,,,,,,,
+97de94dd-7fc6-414b-afbe-934d6489e10e,,,Orthopnea,Orthopnée,Orthopnea,Orthopnée,,,Misc,N/A,,,,,,,,,,,
+063cb6f4-f54d-4ee6-878b-6c683cacac8a,,,Platypnea,Platypnée,Platypnea,Platypnée,,,Misc,N/A,,,,,,,,,,,
+7629a17f-9fc1-4fb6-a766-059cc88620a3,,,Productive Cough,Expectoration,Productive Cough,Expectoration,,,Misc,N/A,,,,,,,,,,,
+78d7751f-5ff3-4cef-b9a7-4c81d55fa5ed,,,Respiratory,Respiratoire,Respiratory,Respiratoire,,,Misc,Coded,,Toux (R05);Hemoptysie (R04.2);Dyspnee (R06.0);Orthopnée;Platypnée;Expectoration;Autre,,,,,,,,,
+97c382db-646b-4ee8-8bc4-fb630a9fc1e6,,,Productive Cough Details,Détails de la toux productive,Productive Cough Details,Détails de la toux productive,,,Misc,Text,,,,,,,,,,,
+1885d33d-9384-4b41-a2a4-4550fea7bf2d,,,Respiratory Set,Respiratoire set,Respiratory Set,Respiratoire set,,,ConvSet,N/A,,,Respiratoire;Détails de la toux productive;Autres observations,,,,,,,,
+00e0aad9-7fa0-48af-9000-003bfd616875,,,Paroxysmal Nocturnal Dyspnea,Dyspnée paroxystique Nocturne,Paroxysmal Nocturnal Dyspnea,Dyspnée paroxystique Nocturne,,,Misc,N/A,,,,,,,,,,,
+25ec3417-f125-4e8c-8862-7f50123b4f81,,,Dyspnea with Exertion,Dyspnée d'Effort,Dyspnea with Exertion,Dyspnée d'Effort,,,Misc,N/A,,,,,,,,,,,
```

Comment:
Dyspnée d'effort

Reply:
Actually I'm not sure that the right translation. Here the french name suggests the dyspnea is happening on phyiscal effort. Which I don't think "with Exertion" means.

Reply:
With exertion implies when the person does any physical work, or movement